web/src/components/profile/ProfileNewJerseyAddress.tsx

```diff
@@ -34,7 +38,7 @@ export const ProfileNewJerseyAddress = (): ReactElement => {
   const { setIsValid: setIsValidZipCode } = useFormContextFieldHelpers("addressZipCode", ProfileFormContext);
 
   useMountEffectWhenDefined(() => {
-    if (business && isOwningBusiness(business)) {
+    if (business && (isStartingBusiness(business) || isOwningBusiness(business))) {
```

Comment:
I'm wondering if we even need to check starting/owning here.

Reply:
Good catch...thank you!